renderer/src/components/RepositorySection/RepositorySection.spec.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { instance, mock, when } from 'ts-mockito';
+import { RepositorySection } from '.';
+import {
+  PullRequest,
+  PullRequestStatus,
+  RepositoryData,
+  User,
+} from '../../models';
+
+describe('RepositorySection', () => {
+  const createPullRequest = ({ status }: { status: PullRequestStatus }) => {
+    const authorMock = mock<User>();
+    when(authorMock.avatarUrl).thenReturn(
+      'https://example.com/images/avatar.jpg'
+    );
+    when(authorMock.name).thenReturn('test');
+    const pullRequestMock = mock<PullRequest>();
+    when(pullRequestMock.status).thenReturn(status);
+    when(pullRequestMock.url).thenReturn(
+      'https://github.com/higeOhige/review-cat/pull/84'
+    );
```

Comment:
テストは通るのでこのままで良いのですが、同じ URL を使い回すので「同じ key があるよ」とエラー出てました 🔍 

```tsx
<li
  key={pullRequest.url ?? pullRequest.title ?? i}
  className={styles.listItemStyle}
>
```

renderer/src/components/RepositorySection/RepositorySection.tsx:37 ですね。

<details>
<summary>
ログ内容
</summary>

```
    console.error
      Warning: Encountered two children with the same key, `https://github.com/higeOhige/review-cat/pull/84`. Keys should be unique so that components maintain their identity across updates. Non-unique keys may cause children to be duplicated and/or omitted — the behavior is unsupported and could change in a future version.
          at ul
          at section
          at RepositorySection (/Users/ynzy/playground/review-cat/renderer/src/components/RepositorySection/RepositorySection.tsx:15:54)
```
</details>

ログ出さないなら `status` を URL の一部にすると良さそうです。

```ts
`https://github.com/higeOhige/review-cat/pull/${status}`
```

Reply:
あ〜
ありがとうございます！！😺
修正しておきやす

Reply:
こちら修正しておきましたー
4f2b610

Reply:
該当のログが出なくなったことを確認しましたー 👍 
ありがとうございます！